packages/legacy/core/App/screens/AttemptLockout.tsx

```diff
@@ -109,32 +111,42 @@ const AttemptLockout: React.FC = () => {
   }, [dispatch, state.loginAttempt.loginAttempts])
 
   return (
-    <SafeAreaView style={styles.container}>
+    <SafeAreaView style={styles.container} edges={['bottom', 'left', 'right']}>
       <ScrollView showsVerticalScrollIndicator={false} scrollEnabled={fontScale >= 1.7}>
         <Assets.svg.appLockout style={styles.image} />
         <ThemedText variant="headingThree" style={styles.title}>
           {t('AttemptLockout.Title')}
         </ThemedText>
         <ThemedText style={styles.description}>{t('AttemptLockout.Description')}</ThemedText>
-        {timeoutDone ? (
-          <Button
-            title={t('Global.TryAgain')}
-            buttonType={ButtonType.Primary}
-            testID={testIdWithKey('Enter')}
-            accessibilityLabel={t('Global.TryAgain')}
-            onPress={unlock}
-          />
-        ) : (
-          <View>
-            <ThemedText style={styles.tryAgain}>{t('AttemptLockout.TryAgain')}</ThemedText>
-            {time && (
-              <ThemedText variant="bold" style={styles.countDown}>
-                {time?.hours} {t('AttemptLockout.Hours')} {time?.minutes} {t('AttemptLockout.Minutes')} {time?.seconds}{' '}
-                {t('AttemptLockout.Seconds')}
-              </ThemedText>
-            )}
+        <View style={styles.actionContainer}>
```

Comment:
This `View` wrapper is just a convenient way to have the same bottom margin regardless of the conditional